lib/pbench/server/api/resources/__init__.py

```diff
@@ -2031,67 +2078,54 @@ def _dispatch(
         # wants to emit a special audit sequence it can disable "finalize"
         # in the context. It can also pass "attributes" by setting that
         # field.
-        auditing = {
-            "audit": audit,
-            "finalize": bool(audit),
```

Comment:
We've lost the conditional setting of `"finalize"`...does that matter?  (You could, for instance, restore this if you opted to use the `__post_init__()` function that I mentioned above.)